include/modem/lte_lc.h

```diff
@@ -150,26 +150,35 @@ enum lte_lc_func_mode {
 
 enum lte_lc_evt_type {
 	/** @brief Event received carrying information about the modems network registration
-	 *         status.
-	 *         Payload is of type @ref lte_lc_nw_reg_status.
+	 *         status. The event type has no associated payload.
```

Comment:
Shouldn't this be the `nw_reg_status` member of type `lte_lc_nw_reg_status`?

Reply:
It sure should, updated now